### DISCUSSION TITLE
[V5] Check whether an input is an array of files or a single one.

### DISCUSSION THREAD

**Discussion by lucasKuratani** [adonisjs/core#1780] [Help]
I need to validate a request that accepts single and multiple file uploads. The problem appears when I try to upload a single file using an array in the schema. The schema is the one that follows.

```    
    const dataSchema = schema.create({
      title: schema.string(),
      subtitle: schema.string.optional(),
      content: schema.string.optional(),
      pictures: schema.array.optional().members(
        schema.file({
          extnames: ['jpeg', 'jpg', 'png', 'gif'],
        }),
      ),
    })
```    

When I try to upload no files, it works. When I try to upload multiple files, it works. When I try to upload a single file, It does not work. It makes sense to me, but I would like to know if there is a way around it.

![Screenshot_1](https://user-images.githubusercontent.com/62770240/96726483-6a9baf80-1388-11eb-81a8-4786b9c11d22.png)

**Comment by thetutlage** [comment]
Can you share the HTML for the file input?
